jsonmodels/fields.py

```diff
@@ -365,6 +365,10 @@ def __init__(self, field: BaseField, *args, **kwargs):
             *args, **kwargs,
         )
 
+    def _finish_initialization(self, owner):
```

Comment:
Is that possible to put a docstring here and indicate why this method is added?
And perhaps also a docstring for the LazyDerivedList test to describe what kind of issue do we run into?

Reply:
Done!